amypad/rectrim.py

```diff
@@ -0,0 +1,44 @@
+import logging
+from pathlib import Path
+
+from niftypet import nimpa
+
+log = logging.getLogger(__name__)
+
+
+def run(
+    dyndir,
+    glob="*itr5*dyn_i.nii*",
+    scale=2,
+    memlim=True,
+    refim="",
+    fcomment="",
+    store_img_intrmd=False,
+    store_img=False,
+):
+    """Trimming and upscaling
+    Args:
+      dyndir: folder containing dynamic scans
+      glob: matching pattern for folder contents
+    """
+    dyndir = Path(dyndir)
+    assert dyndir.is_dir()
+    # get the file data
+    fdyns = list(map(str, dyndir.glob(glob)))
+    log.info("sort the dynamic images and output the sorted file names and numpy array")
+    imdyn = nimpa.niisort(fdyns)
+    log.info("trim & upscale")
+    # NB: the first row is 'blank' due to the offset. TODO: check
+    dtrm = nimpa.imtrimup(
+        imdyn["files"][1:],
```

Comment:
unfortunately doesn't work with example data:

```suggestion
    dtrm = nimpa.imtrimup(
        imdyn["files"],
```